src/components/Mission/Missions.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable arrow-body-style */
+/* eslint-disable no-console */
+import React from 'react';
+import { useSelector } from 'react-redux';
+import Mission from './Mission';
+
+const Missions = () => {
+  const missionStore = useSelector((state) => state.missionReducer);
+  console.log(missionStore);
```

Comment:
`console.log` is good when you are testing your code logic but it should be removed from your production code so that the log does not look messy to the users